packages/pyml/pyml.20210924/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "Thierry Martinez <martinez@nsup.org>"
+homepage: "http://github.com/thierry-martinez/pyml"
+bug-reports: "http://github.com/thierry-martinez/pyml/issues"
+license: "BSD-2-Clause"
+dev-repo: "git+https://github.com/thierry-martinez/pyml.git"
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+run-test: [make "test"]
+synopsis: "OCaml bindings for Python"
+description: "OCaml bindings for Python 2 and Python 3"
+depends: [
+  "ocaml" {>= "3.12.1"}
+  "dune" {>= "2.8.0"}
+  "ocamlfind" {build}
+  "stdcompat" {>= "17"}
+  "conf-python-3-dev" {with-test}
+  "odoc" {with-doc}
```

Comment:
This has already been fixed in the upstream by https://github.com/thierry-martinez/pyml/pull/71.

Reply:
Thank you, @smorimoto ! Is there anything else I should do?

Reply:
Just merged!